source/guides/getting-started.md

```diff
@@ -579,7 +579,7 @@ describe Web::Controllers::Books::Index do
 
   it 'exposes all books' do
     action.call(params)
-    action.exposures[:books].must_equal []
+    action.exposures[:books].must_equal BookRepository.all
```

Comment:
@Bounga This requires a clean database table before the test starts. I'd go for a more explicit way, like the hardcoded array. Example: `[]` or `[book]` where `book` is an entity that we have persisted in the test setup.